Cargo.toml

```diff
@@ -27,12 +27,13 @@ simd_support = [] # enables SIMD support
 serde1 = ["serde", "serde_derive", "rand_core/serde1"] # enables serialization for PRNGs
 
 [workspace]
-members = ["rand_core", "rand_isaac"]
+members = ["rand_core", "rand_isaac", "rand_xorshift"]
 
 [dependencies]
 rand_core = { path = "rand_core", version = "0.2", default-features = false }
 # only for deprecations and benches:
 rand_isaac = { path = "rand_isaac", version = "0.1", default-features = false }
+rand_xorshift = { path = "rand_xorshift", version = "0.1", default-features = false }
```

Comment:
It doesn't have default features (neither does `rand_isaac`).

Reply:
I'll remove it.